### PR TITLE
Slightly changed the page "about" and refactor getWeather function

### PR DESCRIPTION
**Pull request by AndrewHelix**
1. Added more info and background images in About page
2. Refactored getWeather function 
Screenshot 2022-10-11 180901

### CODE REVIEW

File: src/components/about/About.tsx [hunk 1]
```diff
@@ -8,31 +8,48 @@ export const About: FC = () => {
...
       </article>
-      <article className={styles['about-article']}>
+      <article className={classNames(styles['about-article'], styles['about-article__third-item'])}>
         <h3 className={styles['about-article-title']}>Any suggestions or problems?</h3>
-        <p>Please leave them on the feedback page</p>
+        <p>
+          Please leave them on the feedback page. <br />
+          Be sure to leave correct information. You will not be able to leave a review without
+          information about your name, email and phone number. <br />
+          We guarantee that we store your data securely and do not provide it to third parties.{' '}
```
Comment (JuliaShavyrova) [inline src/components/about/About.tsx]: do you need `{' '}`?

File: src/components/main/Forecasts.tsx [hunk 2]
```diff
@@ -24,10 +25,11 @@ const Forecasts: FC = () => {
     state: { position, positionError }
   } = useContext(positionContext);
 
-  async function fetchData(lon: number | string, lat: number | string) {
+  async function fetchData(position: Coordinates) {
+    const { longitude, latitude } = position;
     Promise.all([
-      getCity(lon, lat),
-      getWeather<{ current: CurrentWeatherType }>('/current/', lon, lat)
+      getCity(longitude, latitude),
```
Comment (JuliaShavyrova) [inline src/components/main/Forecasts.tsx]: `getCity` function also can be refactored to use `position`

File: src/components/countries/CitySmallCard.tsx [hunk 3]
```diff
@@ -14,8 +14,13 @@ type SmallCardProps = {
 export const CitySmallCard: FC<SmallCardProps> = ({ lon, lat, cityName }) => {
   const [currentWeather, setCurrentWeather] = useState<CurrentWeatherType | undefined>(undefined);
 
+  const position: Coordinates = {
+    latitude: lat,
+    longitude: lon
+  };
```
Comment (JuliaShavyrova) [inline src/components/countries/CitySmallCard.tsx]: please move variable inside effect body

File: src/components/about/About.tsx (hunk 1 above)
Comment (AndrewHelix) [reply src/components/about/About.tsx]: No, prettier did it)

**Review by JuliaShavyrova — APPROVED** [review]

**Pull request merged**